### PR TITLE
Update generic_pva.xml.fdm_material

### PR DESCRIPTION
**Pull request by maukcc**
Added hardware compatibility

### CODE REVIEW

File: generic_pva.xml.fdm_material [hunk 1]
```diff
@@ -40,8 +40,11 @@ Generic PVA profile. The data in this file may not be correct for your specific
            <setting key="standby temperature">160</setting>
            <setting key="print cooling">0.0</setting>
            <hotend id="0.25 mm" />
+               <setting key="hardware compatible">yes</setting>
            <hotend id="0.4 mm" />
+               <setting key="hardware compatible">yes</setting>
            <hotend id="0.8 mm" />
+               <setting key="hardware compatible">yes</setting>
```
Comment (Ghostkeeper) [inline generic_pva.xml.fdm_material]: I think you wanted these to be put inside the `hotend` tags, but the `hotend` tags close themselves with the `/>` suffix, so these `setting` tags are all in the same `machine` parent tag now and this makes (two of) them duplicates.

Comment (Ghostkeeper) [inline generic_pva.xml.fdm_material]: I think you wanted these to be put inside the `hotend` tags, but the `hotend` tags close themselves with the `/>` suffix, so these `setting` tags are all in the same `machine` parent tag now and this makes (two of) them duplicates.

Comment (maukcc) [inline generic_pva.xml.fdm_material]: Yep, you are right. Changed it.

**Comment by maukcc** [comment]
I made some more changes to it.
Do I need to make a new pull request?

**Comment by Ghostkeeper** [comment]
No, it updates this pull request automatically. I'll review the changes now.

**Pull request merged**